packages/slate/src/models/change.js

```diff
@@ -48,7 +48,7 @@ class Change {
     const { value } = attrs
     this.value = value
     this.operations = new List()
-    this.flags = pick(attrs, ['merge', 'save'])
+    this.flags = pick(attrs, ['merge', 'save', 'normalize'])
   }
```

Comment:
the normalize flag can be set with `change.setOperationFlag()` or in the constructor. This is dangerous for several reasons so it remains an undocumented feature. We are hoping that `withMutations` will be a pleasant enough interface that people will not be tempted to use `change.setOpeartionFlag` for this purpose anyway.